docs/app/pages/components/components-sections/select/select/snippets/validation.html

```diff
@@ -0,0 +1,23 @@
+<form [formGroup]="formGroup">
+    <div class="form-group">
+        <ux-checkbox formControlName="checkbox">Activate select</ux-checkbox>
+    </div>
```

Comment:
Remove the checkbox to simplify the example, the intent is to show how to apply validation to the select component only.